plugins/modules/purefa_pod.py

```diff
@@ -607,7 +607,7 @@ def stretch_pod(module, array):
             if not module.check_mode:
                 try:
                     array.delete_pods_arrays(
```

Comment:
delete_pods_arrays should be changed to delete_pods_members.
Need to handle backward compatibility for older version array.